### PR TITLE
Add a docs-theme table for rendering methods

### PR DESCRIPTION
**Pull request by alfatimfisher**
#### Checklist

- [ ] Includes tests
- [ ] Update documentation

#### Changes proposed in this pull request:

Add a table to docs-theme render methods

Use a @method tag to render methods

#### Reviewers should focus on:

Similar functionality to the interface table but for methods

#### Screenshot

Before:

Screen Shot 2020-05-22 at 11 19 17

After:

Screen Shot 2020-05-22 at 11 47 03

### CODE REVIEW

File: packages/docs-theme/src/components/modifierTable.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Copyright 2020 Palantir Technologies, Inc. All rights reserved.
```
[the remark this answers is not in the document]
Comment (alfatimfisher) [reply packages/docs-theme/src/components/modifierTable.tsx]: Reverted these thanks

**Comment by alfatimfisher** [comment]
> Thanks for the PR! Can you add one usage of this new feature to docs app so that it's actually used in production and we can test the results easily?

@adidahiya I have added some examples for the Table methods https://32525-71939872-gh.circle-artifacts.com/0/packages/docs-app/dist/index.html#table/api.instance-methods

File: packages/docs-data/compile-docs-data.js [hunk 2]
```diff
@@ -80,8 +80,7 @@ function transformDocumentalistData(key, value) {
         // reverse the list so highest version is first (easier indexing)
         return Array.from(majors.values()).reverse();
     }
-    // remove all class declarations as they're currently unused and only increase bundle size
-    if (value != null && value.kind !== "class") {
```
Comment (alfatimfisher) [inline packages/docs-data/compile-docs-data.js:84]: I've had to change this to include the class members but there might be a more efficient way to do it?

**Comment by adidahiya** [comment]
[PR preview docs-app build](https://32525-71939872-gh.circle-artifacts.com/0/packages/docs-app/dist/index.html#table/api.instance-methods)

File: packages/docs-data/compile-docs-data.js (hunk 2 above)
Comment (adidahiya) [reply packages/docs-data/compile-docs-data.js:84]: I think this is fine

**Pull request merged**